.vscode/launch.json

```diff
@@ -0,0 +1,27 @@
+{
+  // Use IntelliSense to find out which attributes exist for node debugging
+  // Use hover for the description of the existing attributes
+  // For further information visit https://go.microsoft.com/fwlink/?linkid=830387
+  "version": "0.2.0",
+  "configurations": [
+    {
+      "name": "Launch Program",
+      "type": "node2",
+      "request": "launch",
+      "program": "${workspaceRoot}/server/dist/server.js",
+      "cwd": "${workspaceRoot}",
+      "sourceMaps": true,
+      "outFiles": [
+        "${workspaceRoot}/server/dist"
+      ]
+    },
+    {
+      "name": "Attach to Process",
+      "type": "node2",
+      "request": "attach",
+      "port": 5858,
+      "outFiles": [],
+      "sourceMaps": true
+    }
+  ]
+}
```

Comment:
Okay. Could you please set your tabSize to 2 and if you add new dependencies to package.json please use ~ instead of ^. Then our npm installs are not to different. ~ looks for the newest patch version and ^ looks for the newest minor & patch version. Maybe its a good idea to use https://docs.npmjs.com/cli/shrinkwrap that we have the same package versions installed.